### PR TITLE
make spellcheck green again

### PR DESCRIPTION
**Pull request by ordian**

### CODE REVIEW

File: node/core/provisioner/src/disputes/prioritized_selection/mod.rs [hunk 1]
```diff
@@ -55,8 +55,8 @@ pub const MAX_DISPUTE_VOTES_FORWARDED_TO_RUNTIME: usize = 200;
 /// it's hard to predict this number because we can't guess how many new votes (for the runtime) a
 /// batch will contain.
 ///
-/// The value below is reached by: `MAX_DISPUTE_VOTES_FORWARDED_TO_RUNTIME` / 2 + 10%
-/// The 10% makes approximately means '10% new votes'. Tweak this if provisioner makes excessive
+/// The value below is calculated by: `MAX_DISPUTE_VOTES_FORWARDED_TO_RUNTIME` / 2 * 1.1%
+/// The `* 1.1%` approximately means 1.1% new votes. Tweak this if provisioner makes excessive
```
Comment (ordian) [inline node/core/provisioner/src/disputes/prioritized_selection/mod.rs:59]: @tdimitrov not quite sure about these numbers and the meaning of `/ 2`. Did you mean something else?

**Review by kianenigma — APPROVED** [review]

**Comment by bkchr** [comment]
How did this even happen? Did someone merged with the test failing?

**Review by sandreim — APPROVED** [review]

**Comment by ordian** [comment]
> How did this even happen? Did someone merged with the test failing?

continuous-integration/gitlab-spellcheck — {"build_allow_failure":**true**} 
but #5567 was merged manually anyway

**Comment by tdimitrov** [comment]
> but https://github.com/paritytech/polkadot/pull/5567 was merged manually anyway

What's the correct way to merge?

File: node/core/provisioner/src/disputes/prioritized_selection/mod.rs (hunk 1 above)
Comment (tdimitrov) [reply node/core/provisioner/src/disputes/prioritized_selection/mod.rs:59]: The explanation is pointless and on top of it the comment itself is wrong.

**Comment by ordian** [comment]
> > but #5567 was merged manually anyway
> 
> What's the correct way to merge?

The correct way is `bot merge`, but it would have ignored this check. You can see the check was red before merge https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/1849069. I don't think we should make it mandatory, fmt check is also ` {"build_allow_failure":true}`.

**Comment by tdimitrov** [comment]
Yes, I saw the job is failing but I ignored it because the errors were mainly in variable names, etc. I'll keep this in mind for the future.

File: node/core/provisioner/src/disputes/prioritized_selection/mod.rs (hunk 1 above)
Comment (tdimitrov) [reply node/core/provisioner/src/disputes/prioritized_selection/mod.rs]: I've pushed the rewording too.

**Review by tdimitrov — APPROVED** [review]

**Comment by ordian** [comment]
bot merge

**Pull request merged**